packages/downloader/download.py

```
@@ -0,0 +1,126 @@
+        
+        # Write to temporary file first
+        with open(temp_path, 'w') as wo:
+            json.dump(data, wo)
+        # Atomically move to final location
+        os.rename(temp_path, final_path)
+    except Exception:
+        # Clean up temp file if it exists
+        if os.path.exists(temp_path):
+            os.remove(temp_path)
+
+def postprocessor_hook(data):
```

Ruff (0.12.2)

Missing return type annotation for public function `postprocessor_hook`

Add return type annotation: `None`

(ANN201)